remo/dashboard/templates/dashboard_reps.jinja

```diff
@@ -1032,6 +1033,9 @@
                       {{ rep|get_display_name }}
                     </a>
                   </td>
+                  <td>
+                    {{ rep.userprofile.country }}
```

Comment:
Is there any chance that a user can have this blank?

Reply:
no is a required field inside the profile